adapters/ddf/src/main/java/com/connexta/replication/adapters/ddf/DdfNodeAdapterFactory.java

```diff
@@ -96,23 +107,29 @@ public ReplicationStore createReplicatorStore(URL url) {
     cswConfiguration.setUsePosList(USE_POS_LIST);
     cswConfiguration.setRegisterForEvents(REGISTER_FOR_EVENTS);
 
-    HybridStore hybridStore =
-        new HybridStore(
+    DdfNodeAdapter ddfNodeAdapter =
+        new DdfNodeAdapter(
             bundleContext,
             cswConfiguration,
             provider,
             clientFactoryFactory,
             encryptionService,
+            ddfRestClientFactory,
             url);
-    hybridStore.setFilterBuilder(filterBuilder);
-    hybridStore.setFilterAdapter(filterAdapter);
-    hybridStore.setResourceReader(resourceReader);
-    hybridStore.setSecurityManager(securityManager);
-    hybridStore.setSchemaTransformerManager(metacardTransformerManager);
-    hybridStore.setCswTransactionWriter(cswTransactionWriter);
-    hybridStore.init();
-
-    return hybridStore;
+    ddfNodeAdapter.setFilterBuilder(filterBuilder);
+    ddfNodeAdapter.setFilterAdapter(filterAdapter);
+    ddfNodeAdapter.setResourceReader(resourceReader);
+    ddfNodeAdapter.setSecurityManager(securityManager);
+    ddfNodeAdapter.setSchemaTransformerManager(metacardTransformerManager);
+    ddfNodeAdapter.setCswTransactionWriter(cswTransactionWriter);
+    ddfNodeAdapter.init();
+
+    return ddfNodeAdapter;
+  }
+
+  @Override
+  public NodeAdapterType getType() {
+    return NodeAdapterType.DDF;
   }
 
   public void setBundleContext(BundleContext bundleContext) {
```

Comment:
❗️ Caching a bundle context tends to create problem as it can get invalidated when the bundle is refreshed or otherwise. It is safer to get it back each time you need it.